packages/preview2-shim/lib/nodejs/http.js

```diff
@@ -179,32 +179,59 @@ class RequestOptions {
   #connectTimeout;
   #firstByteTimeout;
   #betweenBytesTimeout;
```

Comment:
Making the value a top-level constant, we can then set the default value here - `#connectTimeout = defaultTimeoutMs`

Reply:
Updated in https://github.com/bytecodealliance/jco/pull/404/commits/dcf0f31934b9b81b560a7190bc8c0fae8608223f